applications/tari_base_node/src/parser.rs

```diff
@@ -1076,6 +1080,20 @@ impl Parser {
                             }
                             s.join(", ")
                         };
+                        let base_node_data_str = {
+                            let mut s = Vec::new();
+                            match peer.get_metadata(1) {
```

Comment:
Add an enum in tari_core to represent the keys
```rust
#[repr(u8)]
enum PeerMetadata {
   ChainMetadata = 1
}
```

```suggestion
                            match peer.get_metadata(PeerMetadata::ChainMetadata) {
```